src/features/hotspots/details/HexBadge.tsx

```diff
@@ -40,8 +42,9 @@ const HexBadge = ({
           text: t('generic.readMore'),
           style: 'cancel',
           onPress: () => {
-            if (Linking.canOpenURL(Articles.Reward_Scaling))
-              Linking.openURL(Articles.Reward_Scaling)
+            const url = `https://app.hotspotty.net/hotspots/${hotspotId}/reward-scaling`
```

Comment:
Any reason this link can't just be in the single centralized location?

Reply:
It can I suppose but then it would be a function accepting the hotspot id. I also don't see any other place in the app that would need this url in the future, so I didn't consider it tbh. Do you have any particular reasons in favor of moving it to another place?